src/recordings-store.ts

```diff
@@ -1,11 +1,13 @@
 export interface Recording {
-  url: string;
+  url?: string;
+  transcriptUrl?: string;
   roomName: string;
   createdAt: number;
-  ttl: number;
   expiresAt: number;
 }
 
+export const RECORDING_TTL = 86400;
```

Comment:
nit: prefer e.g. `24 * 60 * 60` so I don't have to get a calculator out to understand how long this value is. Also would prefer this constant to have the unit in it as millisecond vs second vs minute confusion has tripped me up many times. e.g. `RECORDING_TTL_SECS`.

But making this a constant is much better than where I put it embedded insider the code somewhere :)

Reply:
agreed: better to let JS do the math (once) and to indicate the units in the constant's name.